entry_test.go

```diff
@@ -22,7 +22,7 @@ func TestEntryWithError(t *testing.T) {
 
 	logger := New()
 	logger.Out = &bytes.Buffer{}
-	entry := NewEntry(logger)
+	entry := NewEntry(logger, 1)
```

Comment:
There is no NewEntry(logger) anymore. You need to feed a number to it like NewEntry(logger, 1). When you set SetCaller(false), the caller will have no effect.